lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -568,9 +567,9 @@ sub update_status ($self) {
         $worker->seen;
         $ret = $job->update_status($status);
     }
-    catch {
+    catch ($error) {
         # uncoverable statement
-        my $error_message = $_;
+        my $error_message = $error; #TOD refactor
```

Comment:
?

Reply:
there are parts i want to come back. they look that can modified accordingly

Reply:
I think `TOD` is a typo of `TODO` and since this is a draft pr I dont see problem on leaving it there while work in progress